apps/example-tailwind/src/app/pages/page-forgot-password/page-forgot-password.component.ts

```diff
@@ -1,14 +1,64 @@
-import { Component } from '@angular/core'
-import { CommonModule } from '@angular/common'
-import { RouterModule } from '@angular/router'
+import { Component, OnInit } from '@angular/core'
+import {
+    FormBuilder,
+    FormGroup,
+    ReactiveFormsModule,
+    Validators,
+} from '@angular/forms'
+import { Router, RouterModule } from '@angular/router'
 import { HlmInputDirective } from '@spartan-ng/ui-input-helm'
 import { HlmCardDirective } from '@spartan-ng/ui-card-helm'
+import { AuthApiService } from '@myorg/common-auth'
+import { CommonModule } from '@angular/common'
 
 @Component({
     selector: 'app-page-forgot-password',
     standalone: true,
-    imports: [CommonModule, RouterModule, HlmInputDirective, HlmCardDirective],
+    imports: [
+        ReactiveFormsModule,
+        RouterModule,
+        CommonModule,
+        HlmInputDirective,
+        HlmCardDirective,
+    ],
     templateUrl: './page-forgot-password.component.html',
     styleUrl: './page-forgot-password.component.scss',
 })
-export class PageForgotPasswordComponent {}
+export class PageForgotPasswordComponent implements OnInit {
+    forgotPasswordForm!: FormGroup
```

Comment:
Do not use `non-null-assertion = !` unless no other way. Here you can directly make the form so this `!` is not required:

```
forgotPasswordForm = new FormGroup({ ... })
```